metadata/metadata.go

```diff
@@ -163,8 +163,8 @@ func AppendToOutgoingContext(ctx context.Context, kv ...string) context.Context
 	return context.WithValue(ctx, mdOutgoingKey{}, rawMD{md: md.md, added: added})
 }
 
-// FromIncomingContext returns the incoming metadata in ctx if it exists.  The
-// returned MD should not be modified. Writing to it may cause races.
+// FromIncomingContext returns the incoming metadata in ctx if it exists.
```

Comment:
Those line layouts are not arbitrary, they are wrapped at 80 characters.

Reply:
Oh,got it.